app/models/manageiq/providers/openstack/cloud_manager/refresh_parser.rb

```diff
@@ -406,8 +406,8 @@ def clean_up_extra_flavor_keys
       end
     end
 
-    def add_instance_disk(disks, size, location, name)
-      super(disks, size, location, name, "openstack")
+    def add_instance_disk(disks, size, location, name, controller_type)
+      super(disks, size, location, name, controller_type)
```

Comment:
so this seems like a incompatible change, there was hardcoded 'openstack' before, now there is a Vm name?